src/components/TableView/TableView.tsx

```diff
@@ -69,12 +68,14 @@ class TableView extends React.Component<Props> {
         {!withoutHeader && !!header && (
           <TableViewHeader header={header} style={headerStyle} />
         )}
-        {React.Children.map(children, (child, idx) =>
-          React.cloneElement(child, {
-            first: idx === 0,
-            last: idx === React.Children.count(children) - 1,
-          })
-        )}
+        {React.Children.map(children, (child, idx) => {
+          if (React.isValidElement(child)) {
```

Comment:
Also need to `return null` if condition didn't satisfy

Reply:
good spot!